CMakeLists.txt

```
@@ -28,6 +28,33 @@ add_executable(accft ${SOURCE})
 set(LIBRARIES fmt::fmt pthread dl m)
 target_link_libraries(accft PUBLIC ${LIBRARIES})
 
+if (SKBUILD)
```

Is `SKBUILD` a standard name? Otherwise I would for something more self explanatory

---

That is the flag set by the build environment scitkit-build to indicate that it has triggered the process.

---

If I understand correctly, this is mainly meant for automatic integration with Scikit rather than something a user would explicitly enable when building the repo. Is that right?

---

Would it make sense to write something like this?

```suggestion
if (SKBUILD)
    set(PYTHON_BINDINGS ON)
endif()

if (PYTHON_BINDINGS)
    message(STATUS "PYTHON_BINDINGS: ${PYTHON_BINDINGS}")
    [...]
```

This way, both the CMake option and `SKBUILD` would work.

---

Yes, you wouldn't build this yourself as you also have to build the python package. You suggestion looks good, we could do it that way.